dist/gmusic.js

```diff
@@ -165,6 +165,15 @@ GMusic.Playback = {
   ALL_SHUFFLE: 'ALL_SHUFFLE',
```

Comment:
On second thought, it's not _that_ bad. I think I'll just implement this and hope for the best.